ee/api/billing.py

```diff
@@ -232,6 +232,34 @@ def get_invoices(self, request: Request, *args: Any, **kwargs: Any) -> HttpRespo
             },
             status=status.HTTP_200_OK,
         )
+    
+    @action(methods=["GET"], detail=False, url_path="credits/eligibility")
+    def credits_eligibility(self, request: Request, *args: Any, **kwargs: Any) -> HttpResponse:
+        license = get_cached_instance_license()
+        if not license:
+            return Response(
+                {"success": True},
+                status=status.HTTP_200_OK,
+            )
+
+        organization = self._get_org_required()
+
+        res = BillingManager(license).credits_eligibility(organization)
+        return Response(res, status=status.HTTP_200_OK)
+    
+    @action(methods=["POST"], detail=False, url_path="credits/purchase")
+    def purchase_credits(self, request: Request, *args: Any, **kwargs: Any) -> HttpResponse:
+        license = get_cached_instance_license()
+        if not license:
+            return Response(
+                {"success": True},
+                status=status.HTTP_200_OK,
+            )
```

Comment:
Seems kinda weird to return a 200 if there is no license

Reply:
Agree, don't want to change that here but I'll change that on another PR, it's the current pattern for all billing endpoints.